cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
```

Comment:
I have a strong aversion to package objects starting to accumulate stuff. 

Unless there's a good reason not to (and others, including whomever made the original request can chime in) I'd prefer to see these in a separate object. I don't really care what the name is.